pandas/core/arrays/interval.py

```diff
@@ -943,7 +943,7 @@ def mid(self):
         else False
         """
 
-    @property
+    @property  # type: ignore # Mypy does not support decorated properties
```

Comment:
Seems to be the state of things: https://github.com/python/mypy/issues/1362

Reply:
Can you put link to Mypy issue here in comment instead (suggested practice from https://mypy.readthedocs.io/en/latest/cheat_sheet.html?highlight=type%20ignore#when-you-re-puzzled-or-when-things-are-complicated)